chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -225,14 +225,32 @@
           repos = Provisioner::Repositories.get_repos(node, "suse", target_platform_version)
           Chef::Log.info("repos: #{repos.inspect}")
 
+          packages = node[:provisioner][:packages][os] || []
+          packages.push("autoyast2-installation")
+          packages.push("biosdevname")
+          packages.push("netcat")
+          packages.push("ruby2.1-rubygem-chef")
+
           if node[:provisioner][:suse]
             if node[:provisioner][:suse][:autoyast]
               ssh_password = node[:provisioner][:suse][:autoyast][:ssh_password]
               append << "UseSSH=1 SSHPassword=#{ssh_password}" if ssh_password
             end
-          end
 
-          packages = node[:provisioner][:packages][os] || []
+            if node[:provisioner][:suse][:cloud_available]
+              packages.push("supportutils-plugin-susecloud")
+
+              if target_platform_version == "11.3"
+                packages.push("use-sle11-openstack-cloud-release")
+              else
+                packages.push("suse-openstack-cloud-release")
+              end
+            end
+
+            if node[:provisioner][:suse][:storage_available]
```

Comment:
So I don't think the logic is correct. If Cloud and Storage are available, we don't want to install ses-release automatically.